torch/csrc/jit/python_tracer.cpp

```diff
@@ -63,6 +64,8 @@ std::shared_ptr<torch::jit::Graph> createGraphByTracing(
     tracer::exit(toStack(out));
     auto graph = enter_info.first->graph;
     EliminateDeadCode(graph);
+    LowerSimpleTuples(graph);
```

Comment:
Is this necessary to reduce the jitter? If yes, please add a comment

Reply:
yes, the tracer and the script do different degrees of initial tuple lowering. I will add a comment to the next pr stacked on this one.